### PR TITLE
Scheme option

### PR DESCRIPTION
**Pull request by mapsam**
Adds an option to pass in a tiling scheme instead of hard coding for specific types. This preserves the defaults that existed before if no option is passed.

This is an alternative to #89 - and would only need a minor release.

cc @GretaCB @rclark

### CODE REVIEW

File: lib/tilelivecopy.js [hunk 1]
```diff
@@ -7,11 +7,12 @@ var combiner = require('stream-combiner');
 var MigrationStream = require('./migration-stream');
 
 function tilelivecopy(srcUri, s3url, options, callback) {
-  var scheme = 'scanline';
+
+  options.type = options.scheme || 'scanline';
+  if(['put', 'pyramid', 'scanline', 'list'].indexOf(options.type) === -1) return callback(new Error('Invalid scheme. Must be "put", "pyramid", "scanline", or "list".'));
```
Comment (rclark) [inline lib/tilelivecopy.js]: I don't think `put` is a valid tilelive scheme for `createReadStream()`

File: readme.md [hunk 2]
```diff
@@ -59,6 +59,11 @@ The following example will distribute tiles to the second part out of 5 total pa
 $ mapbox-tile-copy ~/data/my-tiles.mbtiles s3://my-bucket/parallel/{z}/{x}/{y} --part 1 --parts 4
 ```
 
+Specify a specific tiling scheme to use during copying:
```
Comment (rclark) [inline readme.md]: ```Specify a specific tiling scheme specificity to use specifically during copying```



:trollface:

File: lib/tilelivecopy.js (hunk 1 above)
Comment (mapsam) [inline lib/tilelivecopy.js]: Grabbed it from this list: https://github.com/mapbox/tilelive/blob/master/lib/tilelive.js#L476 but yes you are correct it isn't! Confirmed an error is thrown from tilelive.

File: readme.md (hunk 2 above)
Comment (mapsam) [inline readme.md]: Could you specify what you are specifically saying?

**Comment by mapsam** [comment]
Went ahead with #89 and released 6.0.0 - this PR should still be considered as an added feature to v6

**Comment by mapsam** [comment]
Stale 🍞

**Pull request closed without merging**